### PR TITLE
ABC-348 Review Alembic/Timeline flows

### PR DESCRIPTION
**Pull request by RoseHirigoyen**
**JIRA Ticket:** ABC-348
This PR is to make sure the documentation flows described [here](https://docs.unity3d.com/Packages/com.unity.formats.alembic@2.3/manual/timeline.html) are up to date

### Changes
- Updated the Shot Asset component (+ screenshot) because it seems the offset property was removed (see [this commit](https://github.com/Unity-Technologies/com.unity.formats.alembic/blob/a409a6d50d798cadbd5b476fb5a1cc3e00eedef1/com.unity.formats.alembic/Runtime/Scripts/Timeline/AlembicShotAsset.cs))
- Changed `temporary track` to `Alembic track` to avoid any confusion
- Specified the window which the instruction was referencing for more clarity
- Added a missing step for the infinite timeline workflow
- Updated the links

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.formats.alembic?pullRequest=533) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.formats.alembic?pullRequest=533) before we can accept your contribution.<br/><hr/>**Rose Hirigoyen** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.formats.alembic?pullRequest=533) it.</sub>

**Review by sebastienduverne — COMMENTED** [review]
Cool :)
Only a few link corrections (from former core docs to the latest corresponding pages in Timeline package docs)

**Review by sebastienduverne — APPROVED** [review]
LGTM

File: com.unity.formats.alembic/Documentation~/time_AlembicShot.md [hunk 1]
```diff
@@ -8,14 +8,12 @@ To create an Alembic Shot:
 
 1. Drag a GameObject with an [Alembic Stream Player](ref_StreamPlayer.md) component onto the Clips view in Timeline.
 
-   If Timeline is empty, create a temporary track so that you can see the Clips view.
+   If Timeline is empty, create an Alembic track so that you can see the Clips view.
```
Comment (cguer) [inline com.unity.formats.alembic/Documentation~/time_AlembicShot.md:11]: I think we can remove this line completely. We don't need to create an empty track for this to work, we can just drag it into the empty space of the timeline. I'm not quite sure why it's written this way but I suspect it was an ealier and more limited version of Timeline.
abcclip_001

**Review by cguer — APPROVED** [review]
I think one sentence can be removed completely but otherwise QA approved 👍!

File: com.unity.formats.alembic/Documentation~/time_AlembicShot.md (hunk 1 above)
Comment (RoseHirigoyen) [reply com.unity.formats.alembic/Documentation~/time_AlembicShot.md]: Removed in [39bc8b8](https://github.com/Unity-Technologies/com.unity.formats.alembic/pull/533/commits/39bc8b88ab949acae1756742f10cdc02ffd35674)

**Pull request merged**